README.md

```diff
@@ -87,22 +107,18 @@ Customize in `~/.laptop.local`
 
 Your `~/.laptop.local` is run at the end of the Laptop script.
 Put your customizations there.
-For example:
+Possible customizations are:
```

Comment:
Let's minimize wording changes that could cause upstream conflicts later. I'm not sure there is a compelling reason to change this.